.github/workflows/ci.yml

```diff
@@ -72,9 +75,9 @@ jobs:
         uses: actions/cache@v2
         with:
           path: vendor/bundle
-          key: ${{ runner.os}}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          key: ${{ runner.os }}-gems-${{ env.ImageVersion }}-${{ hashFiles('**/Gemfile.lock') }}
```

Comment:
Should we do the same for pod caching too?

Reply:
Mayhaps, but this doesn't actually work so 😓